src/app/components/mno-address/mno-address-line/mno-address-line.coffee

```diff
@@ -0,0 +1,14 @@
+@App.directive('mnoAddressLine', ->
+  return {
+    restrict: 'EA'
+    templateUrl: 'app/components/mno-address/mno-address-line/mno-address-line.html',
+    scope: {
+      address: '='
+    }
+
+    controller: ($scope) ->
+      displayAttrs = ['street', 'city', 'state_code', 'postal_code', 'country_code']
+      $scope.displayAddress = _.map(displayAttrs, (attr) -> $scope.address[attr]).join(', ')
```

Comment:
You can use CoffeeScript comprehensions to not have to rely on lodash:
```coffeescript
$scope.displayAddress = ($scope.address[attr] for attr in displayAttrs).join(',')
```

See this for more coffescript idioms: https://arcturo.github.io/library/coffeescript/04_idioms.html